workers/connect_to_postgres.py

```diff
@@ -0,0 +1,14 @@
+import psycopg2
+
+def connection() :
+    try :
+        connection_details = "dbname='makeupdb' user='icebear' host='localhost' port='5432'" 
```

Comment:
Please put the db connection detail values into your env file too before loading it in